### PR TITLE
Fix database not getting set correctly in DacFx wizard deploy scenario

### PR DESCRIPTION
**Pull request by kisantia**
This fixes the database not getting set correctly for deploy scenario if coming from the import page. Previously, the database value was only getting set when the value was undefined, but it is already set to a value when coming a different page before going through the deploy scenario steps.

### CODE REVIEW

File: extensions/import/src/wizard/pages/deployConfigPage.ts [hunk 1]
```diff
@@ -172,7 +172,8 @@ export class DeployConfigPage extends DacFxConfigPage {
 		}
 		let values = await this.getDatabaseValues();
 
-		if (this.model.database === undefined) {
+		// set the database to the first dropdown value if upgrading, otherwise it should get set to the textbox value
```
Comment (alanrenmsft) [inline extensions/import/src/wizard/pages/deployConfigPage.ts]: nit: in ADS repo, comments go directly after //

Comment (kisantia) [reply extensions/import/src/wizard/pages/deployConfigPage.ts]: so it should be like //set... instead of // set... ?

Comment (alanrenmsft) [reply extensions/import/src/wizard/pages/deployConfigPage.ts]: Yes, that's what Karl commented for one of my early PRs, But I did notice there are a lot of places we have a space in between.

Comment (kisantia) [reply extensions/import/src/wizard/pages/deployConfigPage.ts]: ok I changed the comments in this file to remove the space

**Pull request merged**